### DISCUSSION TITLE
Toggle column does not respect update policy?

### DISCUSSION THREAD

**Discussion by rlitvin** [filamentphp/filament#13571] [Help]
### Package

Table builder

### Package Version

v3.2.86

### How can we help you?

I'm using spatie laravel permissions package, is it possible that this toggle column is not checking the update policy from the model??

**Comment by paulknisely** [comment]
Just ran into this myself. Did you ever find a solution?

**Reply by rlitvin** [reply]
No, same happens with text input column...

**Reply by paulknisely** [reply]
Bummer this isn't natively supported. Wish I had the time to learn and contribute a fix.

This was the best 'workaround' that I could think of.

```php
Tables\Columns\ToggleColumn::make('toggle_example')
    ->disabled(fn () => auth()->user()->cannot('toggle values')),
```

**Reply by rlitvin** [reply]
Yes but that's no good. You have to remember every time you make a toggle column.

**Reply by bilogic** [reply]
file a bug
